tests/test_iib_ops.py

```diff
@@ -213,6 +213,56 @@ def add_bundles_mock_calls_tester(
         binary_image="binary-image",
         overwrite_from_index=True,
         overwrite_from_index_token="overwrite_from_index_token",
+        deprecation_list=["bundle1"],
+    )
+    fixture_iib_client.assert_called_once_with(
+        "iib-server", auth=fixture_iib_krb_auth.return_value, ssl_verify=False
+    )
+    fixture_pulplib_repo_sync.assert_called_once()
+    assert fixture_pulplib_repo_sync.mock_calls[0].args[0].feed == "https://feed.com"
+    fixture_pulplib_repo_publish.assert_called_once()
+
+
+def add_bundles_mock_calls_tester_empty_deprecation_list(
```

Comment:
Optional: You can probably use `pytest.parameterize` for this and the test below.

Reply:
I would have to research how to use parametrize of this function in a test that uses parametrize too.

Reply:
I don't understand. You can simply combine this test and `add_bundles_mock_calls_tester_deprecation_bundles` using an approach similar to https://github.com/release-engineering/pubtools-iib/pull/20/files/20668514dc69ca31cabd8f905907a2057d7cb3e9#diff-9c4a86c181a4176bddb1e77c859da5be358901042d39f308d1db4a1eaeb70923R342

Reply:
Yeah, I overlook that this is not test but just  some helper function. Right now I am not sure if can use parameterize (I would say we do not)

Reply:
`add_bundles_mock_calls_tester_empty_deprecation_list` is a mock function which we use bellow at `test_add_bundles_cli` line #399.

Reply:
oh! I missed that too. +1 to keep it separate